addons/mail/static/src/xml/thread.xml

```diff
@@ -57,7 +57,10 @@
                     <div t-if="widget.activeAttachment.fileType == 'image'" class="o_loading_img text-center">
                         <i class="fa fa-circle-o-notch fa-spin text-gray-light fa-3x fa-fw" role="img" aria-label="Loading" title="Loading"/>
                     </div>
-                    <img t-if="widget.activeAttachment.fileType === 'image'" class="o_viewer_img" t-attf-src="/web/image/#{widget.activeAttachment.id}?unique=1&amp;signature=#{widget.activeAttachment.checksum}&amp;model=#{model}" alt="Viewer"/>
+                    <t t-if="widget.activeAttachment.fileType === 'image'">
```

Comment:
cant we ref the t-if with l57?

Does it work in case field image attachment=False... ? probably no checksum...
Or if model is always attachment, why not use string in L62 to avoid ambiguity?

Reply:
For the second part of the comment, we don't have use cases where we'd use a object/record with a binary field with attachment=False. The documentViewer uses `ir.attachment`-like objects.

But I guess it would still work, just not use the cache.